tutorials/roofit/rf615_learned_likelihoods.py

```diff
@@ -0,0 +1,231 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook
+## Use Simulation Based Inference (SBI) in RooFit
```

Comment:
Can you rename this file to `rf615_simulation_based_inference.py` as we discussed? Since the morphing is not really "learning", and it also doesn't approximate the likelihood ration but the pdf value. So I think SBI would be more general (and maybe more catchy too).